tools/gen_ucd_tables.py

```diff
@@ -28,12 +28,12 @@
 from common import path, memoize
 from unicode_utils import is_surrogate
 
-
```

Comment:
Didn't have autopep8 installed previously (don't write too much python and PyCharm/IDEA usually do a decent job of formatting along pep8). Now I do.

Strangely, it's not changing this. I went ahead and changed it back manually.